statsmodels/base/tests/test_generic_methods.py

```diff
@@ -227,28 +403,50 @@ def setup(self):
         x = self.exog
         np.random.seed(987689)
         y_count = np.random.poisson(np.exp(x.sum(1) - x.mean()))
-        model = sm.Poisson(y_count, x)  #, exposure=np.ones(nobs), offset=np.zeros(nobs)) #bug with default
+        model = sm.Poisson(y_count, x)
+        # use start_params to converge faster
+        start_params = np.array([0.75334818, 0.99425553, 1.00494724, 1.00247112])
+        self.results = model.fit(start_params=start_params, method='bfgs',
+                                 disp=0)
+
+
+class TestGenericPoissonOffset(CheckGenericMixin):
+
+    def setup(self):
+        #fit for each test, because results will be changed by test
+        x = self.exog
+        nobs = x.shape[0]
+        np.random.seed(987689)
+        y_count = np.random.poisson(np.exp(x.sum(1) - x.mean()))
+        model = sm.Poisson(y_count, x, offset=0.01 * np.ones(nobs),
+                           exposure=np.ones(nobs)) #bug with default
```

Comment:
Is there a GH label for this "bug with default"?

Reply:
Yes, I opened #4595, when I found this.

Reply:
OK thanks.  If convenient, adding an extra space before or after the `#` here would make this easier for me; you've been very responsive to my requests so far so I don't want to push my luck.